java/src/jmri/jmrit/withrottle/MultiThrottle.java

```diff
@@ -128,6 +128,11 @@ protected void stealThrottleController(String key, String action) {
      * @param key address to be validated, of form Lnnnn or Snnn
      */
     private boolean isValidAddr(String key) {
+        if (key.length() < 2) {
+            String msg = Bundle.getMessage("ErrorAddressTooShort", key);
+            log.warn(msg);
+            return false;
```

Comment:
I'd suggest adding
`parentController.sendAlertMessage(msg);`
so that the client is informed of the error.

Reply:
Thanks. I haved added it.